packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/fresco/FrescoModule.java

```diff
@@ -13,6 +13,7 @@
 import com.facebook.imagepipeline.backends.okhttp3.OkHttpImagePipelineConfigFactory;
 import com.facebook.imagepipeline.core.ImagePipeline;
 import com.facebook.imagepipeline.core.ImagePipelineConfig;
+import com.facebook.imagepipeline.core.ImagePipelineConfig.DownsampleMode;
```

Comment:
I think the import should be changed to the following, as I was unable to run a build with the current one:

```suggestion
import com.facebook.imagepipeline.core.ImagePipeline;
import com.facebook.imagepipeline.core.DownsampleMode;
import com.facebook.imagepipeline.core.ImagePipelineConfig;
```

Reply:
@sunnylqm